<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationClass</class>
 <widget class="QWidget" name="InformationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InformationClass</string>
  </property>
  <widget class="QLabel" name="host_name_label_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Host Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ip_address_label_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Address:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="host_name_line_edit_">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ip_address_line_edit_">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="details_push_button_">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Details</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="detail_text_browser_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>331</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
